src/css/utils/size.scss

```diff
@@ -12,7 +12,7 @@ $responsive: true;
 @each $col in $cols {
   @for $i from 1 through ($col - 1) {
     .u-size#{$i}of#{$col} {
-      flex-basis: auto !important;
```

Comment:
@saralohr wanted to get your thoughts on this change. Was running into issues with u-sizeFit not working in on things like Panel-cell that had the flex: shorthand definition, so just overriding flex-basis wasn't enough.

Reply:
I'd be pretty hesitant to make this change, since the flex shorthand for growing and shrinking can act in unexpected ways, and it would be a divergence from the SUIT CSS system that this is pulled from. Can the Panel-cell flex property be changed instead? Could you share the specific use case of what you are trying to achieve?

Reply:
I am too. The issue with Panel-cell is that it's set to automatically fit the within the row with `flex: 1 1 0%;` (which I wouldn't want to change) and widths can't be defined as anything smaller than flex defines them as. It seems like the content around it can be set wider and it'll in turn shrink the cells around it to accommodate, but I'm not able to set the small columns and let the larger one grow to fit the space left. It would be nice to be able to define the smaller column since it's usually more defined content in terms of size.

Reply:
I think there may be a few existing utility classes that you could add to `Panel-cell` to get around this. Size utilities should work to override the flex-basis, or even `u-flexInitial` which sets the flex shorthand as `0 1 auto` so it won't grow to fill the space. We could also add a class `u-flexGrow0`. If not let's pair and see if we can find a solution.

Reply:
Good call on the u-flexGrow0 utility! I'll do that